test/removeComponentValidFilters.js

```diff
@@ -0,0 +1,15 @@
+import ECS from '../ecs.js'
```

Comment:
please avoid committing package-lock.json changes in the PR

Reply:
Sorry about that! I will fix the PR within few days. (fix the fix;-))